src/inference/src/ie_core.cpp

```diff
@@ -515,6 +515,13 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
             // Take device name if device does not support DEVICE_ARCHITECTURE metric
             compileConfig[ov::device::architecture.name()] = deviceFamily;
         }
+
+        // 3. extract config keys which affect compile config
+        if (DeviceSupportsConfigKey(plugin, ov::device::caching_properties.name())) {
```

Comment:
are they not a part of `origConfig` ?

Reply:
Usually not. If we configure core with device configs and call compile_model without config for one of the devices, the `origConfig` will be empty. E.g. in benchmark_app we usually use exactly this way.

Reply:
Got it, thanks

Two more comments:
- let's have in `compileConfig` those ones from `origConfig`, who are in list `caching_properties`
- compile_model's properties have higher priority with compare to plugin's ones

Reply:
`compileConfig` formation has been changed, thanks